ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/panelcontainer/v1/panelcontainer/responsiveGrid.html

```diff
@@ -29,4 +29,6 @@
         <div data-sly-call="${longDescription.longDescription @componentId=panel.id, longDescription=panel.description, bemBlock='cmp-container'}" data-sly-unwrap></div>
         <sly data-sly-resource="${resource @ resourceType='wcm/foundation/components/responsivegrid'}"></sly>
     </div>
+    <div data-cmp-hook-add-instance> Add me </div>
```

Comment:
Add me?

Reply:
This is just a test code, will be removed shortly, once complete repeatability including accordion and wizard is checked in

Reply:
removed, for now.